docs/api-docs/storefront/graphql/graphql-site-content.mdx

```diff
@@ -0,0 +1,1004 @@
+                    }
+                  ]
+                }
+              }
+            }
+          ]
+        }
+      }
+    }
+  }
+}
+```
+
+</Tab>
+</Tabs>
+
+The following example retrieves rendered widgets on contact form pages.
```

Comment:
This example might be somewhat duplicative, should the reader be able to infer this from the previous example?

Reply:
True, I decided to remove it ✔️